tools/github-event-processor/Azure.Sdk.Tools.GitHubEventProcessor/Constants/CommentConstants.cs

```diff
@@ -0,0 +1,28 @@
+using System;
+using System.Collections.Generic;
+using System.Drawing;
+using System.Text;
+
+namespace Azure.Sdk.Tools.GitHubEventProcessor.Constants
+{
+    /// <summary>
+    /// Strings in comments that cause actions to be included/excluded from processing.
+    /// </summary>
+    public class CommentConstants
+    {
+        // /azp <command> does things like prepares new pipelines or kicks off pipelines and should
+        // also be ignored for comment processing. 
+        public static readonly string Azp = "/azp";
```

Comment:
Why do we care about ignoring these? It isn't ideal to hardcode these here unless we feel there is a real issue we are solving.

Reply:
We care about ignoring these because the particular rule, [Reset Pull Request Activity](https://github.com/Azure/azure-sdk-tools/pull/5607/files#diff-a6f100fefea1bd20c1b586dfef23f235bfd3d0e8589681766011240ddd1c4932R59), shouldn't be processing if the comment is an /azp or /check-enforcer command.